sdk/resourcemanager/azure-resourcemanager-cosmos/src/main/java/com/azure/resourcemanager/cosmos/models/CosmosDBAccount.java

```diff
@@ -544,13 +556,31 @@ interface WithLocalAuth {
             WithCreate disableLocalAuth();
         }
 
+        /**
+         * The stage of CosmosDB account definition allowing to enable automatic failover.
+         */
+        interface WithAutomaticFailover {
+            /**
+             * Disables automatic failover.
+             * Enables automatic failover of the write region in the rare event that the region is unavailable due to an outage.
+             * Automatic failover will result in a new write region for the account and is chosen based on the failover
+             * priorities configured for the account.
```

Comment:
Is this API intend for "enable" or "disable"?

Reply:
Yes, it's for `disable`. Fixed.